app/src/main/java/fr/free/nrw/commons/depictions/models/Searchinfo.java

```diff
@@ -0,0 +1,42 @@
+package fr.free.nrw.commons.depictions.models;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Model class for object obtained while parsing query object
+ */
+
+public class Searchinfo {
```

Comment:
Are we using `totalhits` anywhere?

Reply:
No but it raises the 2 camps of network modelling

1. You should model the object in its entirety to exactly represent it
1. You should only model what you need

I fall into the first camp even though in general I would consider cutting off my own finger if I thought I would never use it again.